src/components/Missions/missions.module.css

```diff
@@ -28,9 +28,10 @@ tr:nth-child(even) {
 .missionReserved {
   text-align: center;
   font-size: 14px;
-  word-spacing: -2px;
+
+  /* word-spacing: -2px; */
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes requested by the previous reviewer :clap: , However as mentioned in the Best practices guidelines we **do not commit old pieces of code as inline comments. they will make our project look messy.** remove all the commented like this line.